### PR TITLE
Allow admins to define onboarding images in config

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This allows admins to define the values for onboarding....

Screen Shot 2020-06-11 at 7 18 38 PM

Addresses a portion of this...

https://github.com/thepracticaldev/dev.to/issues/5507

### CODE REVIEW

File: app/controllers/internal/configs_controller.rb [hunk 1]
```diff
@@ -135,6 +135,8 @@ def image_params
 
   def onboarding_params
```
Comment (Ridhwana) [inline app/controllers/internal/configs_controller.rb:136]: We should also add a test to the spec⁩/⁨requests⁩/internal⁩/configs_spec to test the updating of the       onboarding_logo_image and onboarding_background_image.

File: spec/requests/onboardings_spec.rb [hunk 2]
```diff
@@ -14,5 +14,22 @@
...
       expect(response).to have_http_status(:ok)
     end
+
+    it "contains proper data attribute keys" do
+      sign_in user
+      get onboarding_url
+      expect(response.body).to include("community-description")
+      expect(response.body).to include("community-logo")
+      expect(response.body).to include("community-background")
+      expect(response.body).to include("data-community-name")
```
Comment (Ridhwana) [inline spec/requests/onboardings_spec.rb:24]: Should this not just have been "community-name" like the rest?

Comment (benhalpern) [reply spec/requests/onboardings_spec.rb:24]: ✅

File: app/controllers/internal/configs_controller.rb (hunk 1 above)
Comment (benhalpern) [reply app/controllers/internal/configs_controller.rb:136]: ✅

**Comment by benhalpern** [comment]
@Ridhwana pushed some new tests

**Pull request merged**